modLogger/src/org/aion/log/AionLoggerFactory.java

```diff
@@ -70,12 +75,67 @@
         }
     }
 
+    /** Change INITIALIZE signature to include LOGFILE */
     public static void init(final Map<String, String> _logModules) {
+        init(_logModules, false, "log");
+    }
 
-        logModules = _logModules;
+    // public static void init(final Map<String, String> _logModules) {
+    public static void init(final Map<String, String> _logModules, boolean _logToFile, String _logToFolder) {
 
+        logModules = _logModules;
+        logToFile = _logToFile;
+        logPath = _logToFolder;
         loggerContext = (LoggerContext) LoggerFactory.getILoggerFactory();
 
+        /** Toggles file appending configurations */
+        if (logToFile) {
+            /** Initialize Rolling-File-Appender */
+            String fileName = "./" + logPath + "/aionCurrent.log";
```

Comment:
No need to add the ``./`` in front of logPath, we don't need to confine logs to the local directory